### PR TITLE
Solo Code Review - Task

### PR DESCRIPTION
**Pull request by felixDev22**
- Created a new repo "Solo-Review".
 - Created a new branch "Solo-branch".
- Copied code from the project that I completed during the Admissions Trial.
- Project work properly.
- Checked for linter errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div id="main">
+      <div class="header">
+        <div class="container">
+          <div id="title">
+            <h1>BOB <span> MARLEY</h1>
```
Comment (felixDev22) [inline index.html:21]: Please use the "ul li" tag for links, and logos. It's best HTML practices.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+              perspiciatis vel ad, nihil hic. Corporis laboriosam quis animi
+              rerum libero nisi quod, nemo velit earum dolores. Necessitatibus
+              corrupti provident repudiandae!
+            </p>
+
+            <a
+              href="https://www.bobmarley.com/"
+              target="_blank"
+              id="tribute-link"
+              >Read more</a
```
Comment (felixDev22) [inline index.html:50]: I think it would be great to use the "button" tag for "read more".

File: index.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/css/bootstrap.min.css"
+      integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+      crossorigin="anonymous"
+    />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div id="main">
```
Comment (felixDev22) [inline index.html:17]: Please use the "Main" tag avoid using "div" for sections. Writing Semantic HTML gives your code meaning to web browsers.

**Review by felixDev22 — COMMENTED** [review]
**Changes required**
You are almost there....just a few changes and we are good to go.

**Review by ishimwezachee — COMMENTED** [review]
Hi @felixDev22 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlights
Added inline comments ✔️
Comments are clear and polite ✔️
Explained to a student why suggesting changes✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**